### PR TITLE
Add documentation that fixes incorrect audit log setting names

### LINKED ISSUE #4651: [DOC]Update audit log settings for security
**What do you want to do?**
 
- [ ] Request a change to existing documentation
- [ ] Add new documentation
- [ ] Report a technical problem with the documentation
- [ ] Other 

**Tell us about your request.**
Several audit log settings configured in the `audit.yml` file are expressed incorrectly in documentation. Also, there is no clear guidance in the documentation that informs users that these are initially configured in the `audit.yml` file.
 

**What other resources are available?**

This issue is based on the security repo's [#2673](https://github.com/opensearch-project/security/issues/2673).
The affected documentation page:
[Audit logs](https://opensearch.org/docs/latest/security/audit-logs/index/)

### PR DESCRIPTION
**Pull request by cwillum**
### Description
Several setting names in Audit log documentation were missing the `.config` element. Fixed these setting names.

### Issues Resolved
Fixes #4651 

### Checklist
- [ ] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by cwillum** [comment]
@scrawfor99 Could you have a look at this update to audit log settings and make sure it covers the problems we were having in [#2673](https://github.com/opensearch-project/security/issues/2673) ? Also, does it sound appropriate to backport this documentation to 2.0? This just means documentation versions 2.0 through 2.9 will be updated.
Thanks.

**Comment by scrawfor99** [comment]
Hi @cwillum, no problem. And yeah, that is correct about the versions. I am not sure if the documentation site is still updates all versions or only the active ones. One of the maintainers may be able to let you know about that.

**Comment by cwillum** [comment]
@scrawfor99 RE: versions (doc). We recently decided to keep the most recent documentation version up to date. But we reserved the right to backport if we thought it was a good idea. In this case, given the problems underlying the changes in this PR, I thought it was a good idea to feed those changes back to 2.0.

**Comment by cwillum** [comment]
@scrawfor99 Apologies. Did the last reply suggest that this looks good? Or you still need a minute to review? I reorganized a little for clarity. I'll move this through doc team review and leave you alone if it looks OK. Thanks again.

File: _security/audit-logs/index.md [hunk 1]
```diff
@@ -25,7 +26,9 @@ To enable audit logging:
 
 2. Restart each node.
 
-After this initial setup, you can use OpenSearch Dashboards to manage your audit log categories and other settings. In OpenSearch Dashboards, choose **Security**, **Audit logs**.
+After this initial setup, you can use OpenSearch Dashboards to manage your audit log categories and other settings. In OpenSearch Dashboards, select **Security** and then **Audit logs**. 
+
+An alternative is to make initial settings for audit logging in the `audit.yml` and `opensearch.yml` files (which file depends on the setting---see [Audit log settings](#audit-log-settings)). Thereafter, you can use Dashboards or the [Audit logs]({{site.url}}{{site.baseurl}}/security/access-control/api/#audit-logs) API to manage and update settings.
```
Comment (natebower) [inline _security/audit-logs/index.md:31]: Can we use a more precise verb than "make"?

File: _security/audit-logs/index.md [hunk 2]
```diff
@@ -115,55 +126,69 @@ audit_trace_resolved_indices: [
 You can disable this feature by setting:
 
 ```yml
-plugins.security.audit.resolve_indices: false
+plugins.security.audit.config.resolve_indices: false
 ```
 
-Disabling this feature only takes effect if `plugins.security.audit.log_request_body` is also set to `false`.
+Disabling this feature only takes effect if `plugins.security.audit.config.log_request_body` is also set to `false`.
```
Comment (natebower) [inline _security/audit-logs/index.md:132]:
```suggestion
This feature is only disabled if `plugins.security.audit.config.log_request_body` is also set to `false`.
```

File: _security/audit-logs/index.md [hunk 3]
```diff
@@ -115,55 +126,69 @@ audit_trace_resolved_indices: [
...
 
 The Security plugin can be configured to log each indexing operation as a separate event:
 
 ```yml
-plugins.security.audit.resolve_bulk_requests: true
+plugins.security.audit.config.resolve_bulk_requests: true
 ```
 
-This change can create a massive number of events in the audit logs, so we don't recommend enabling this setting if you make heavy use of the `_bulk` API.
+This change can create an extremely large number of events in the audit logs, so we don't recommend enabling this setting if you make heavy use of the `_bulk` API.
```
Comment (natebower) [inline _security/audit-logs/index.md:146]:
```suggestion
This change can create an extremely large number of events in the audit logs, so we don't recommend enabling this setting if you frequently use the `_bulk` API.
```

File: _security/audit-logs/index.md [hunk 4]
```diff
@@ -115,55 +126,69 @@ audit_trace_resolved_indices: [
...
 
 
-## Exclude requests
+#### Exclude requests
 
-You can exclude certain requests from being logged completely, by either configuring actions (for transport requests) and/or HTTP request paths (REST):
+You can exclude certain requests from being logged completely by configuring actions for transport requests and/or HTTP request paths (REST):
```
Comment (natebower) [inline _security/audit-logs/index.md:151]: This reads as though "completely" should precede either "logged" or "exclude".

File: _security/audit-logs/index.md [hunk 5]
```diff
@@ -115,55 +126,69 @@ audit_trace_resolved_indices: [
...
-plugins.security.audit.ignore_users: NONE
+plugins.security.audit.config.ignore_users: NONE
 ```
 
 
-## Configure the audit log index name
+#### Exclude headers
 
-By default, the Security plugin stores audit events in a daily rolling index named `auditlog-YYYY.MM.dd`. You can configure the name of the index in `opensearch.yml`:
+You can exclude sensitive headers from being included in the logs---for example, the `Authorization:` header.
```
Comment (natebower) [inline _security/audit-logs/index.md:177]:
```suggestion
You can exclude sensitive headers from being included in the logs---for example, the `Authorization:` header:
```

File: _security/audit-logs/index.md [hunk 6]
```diff
@@ -178,16 +203,17 @@ plugins.security.audit.config.index: "'auditlog-'YYYY.MM.dd"
 For a reference on the date pattern format, see the [Joda DateTimeFormat documentation](https://www.joda.org/joda-time/apidocs/org/joda/time/format/DateTimeFormat.html).
 
 
-## (Advanced) Tune the thread pool
+#### (Advanced) Tune the thread pool
 
-The Search plugin logs events asynchronously, which keeps performance impact on your cluster minimal. The plugin uses a fixed thread pool to log events. You can define the number of threads in the pool in `opensearch.yml`:
+The Search plugin logs events asynchronously, which keeps performance impact on your cluster minimal. The plugin uses a fixed thread pool to log events:
```
Comment (natebower) [inline _security/audit-logs/index.md:208]:
```suggestion
The Search plugin logs events asynchronously, which minimizes the performance impact on your cluster. The plugin uses a fixed thread pool to log events:
```

**Review by natebower — APPROVED** [review]
@cwillum Approved with noted changes. Let me know if you have any questions. Thanks!

**Pull request merged**